lib/game.rb

```diff
@@ -28,63 +32,63 @@ def main_menu
     user_input
   end
 
-  def display_boards(user, computer)
-    puts "=============COMPUTER BOARD============="
-    puts computer.render
-    puts "==============PLAYER BOARD=============="
-    puts user.render(true)
-  end
-
-  # def play
-  #   while main_menu == 'p'
-  #     complete_computer_setup
-  #     complete_user_setup
-  #
-  #   end
-  # end
-
   def take_turns(user, computer)
     display_boards(user, computer)
     while winner?(user, computer) == false
-      computer_turn = Turn.new(computer, user, :computer)
-      computer_turn.take_turn(computer, user, :computer)
+      computer_turn = Turn.new(user, computer, :computer)
+      computer_turn.take_turn(user, computer, :computer)
       display_boards(user, computer)
       break if winner?(user, computer) == true
-      user_turn = Turn.new(computer, user, :human)
```

Comment:
Can we add spaces between the display boards?

Reply:
Added!